jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -150,13 +153,18 @@ protected void validateSupportedPropertiesList() throws PropertyValidationExcept
         }
     }
 
+    /**
+     * Get the list of user-defined adapter properties which the dialect supports
+     *
+     * @return list of supported properties
+     */
+    protected abstract List<String> getSupportedProperties();
+
     protected String createUnsupportedElementMessage(final String unsupportedElement, final String property) {
```

Comment:
Unsupported Element? It's only about properties, isn't it?

Reply:
Check lines 186 and 193 to see the meaning.